lib/rules/eol-last.js

```diff
@@ -1,62 +1,95 @@
 /**
- * @fileoverview Require file to end with single newline.
+ * @fileoverview Require or disallow file to end with single newline.
  * @author Nodeca Team <https://github.com/nodeca>
+ * @author kdex <https://github.com/kdex>
  */
 "use strict";
 
 //------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const lodash = require("lodash");
+
+//------------------------------------------------------------------------------
 // Rule Definition
 //------------------------------------------------------------------------------
 
 module.exports = {
     meta: {
         docs: {
-            description: "enforce at least one newline at the end of files",
+            description: "enforce at least one newline (or absence thereof) at the end of files",
```

Comment:
Can we make this consistent with the rule description in the title of the docs file? Thoughts on "require or disallow newlines at the end of files"?